spec/configuration/prison_data_spec.rb

```diff
@@ -20,8 +20,8 @@
   context "enabled prisons" do
     let(:enabled) { Prison.enabled }
 
-    it "currently number between 95 and 105" do
-      expect(enabled.count).to be_within(5).of(100)
+    it "currently number at least 95" do
+      expect(enabled.count).to be >= 95
```

Comment:
I don't know what magic RSpec does internally to make this work, but it does work!